packages/common/src/number.ts

```diff
@@ -0,0 +1,17 @@
+const ranges = [
+  { divider: 1e18, suffix: "E" },
+  { divider: 1e15, suffix: "P" },
+  { divider: 1e12, suffix: "T" },
+  { divider: 1e9, suffix: "G" },
+  { divider: 1e6, suffix: "M" },
+  { divider: 1e3, suffix: "k" },
+];
+
+export const formatNumber = (value: number, decimalPlaces: number) => {
```

Comment:
Suggestion: add a radix input to get this function to get value in kb, mb, gb… (2)

Reply:
I think this should be a different function? Otherwise you violate the SRP https://en.wikipedia.org/wiki/Single-responsibility_principle

Reply:
Yes format number is only to format a number and not for sizes